cachi2/core/package_managers/generic/models.py

```diff
@@ -10,6 +11,8 @@
 from cachi2.core.errors import PackageManagerError
 from cachi2.core.rooted_path import RootedPath
 
+CHECKSUM_FORMAT = re.compile(r"^[a-zA-Z0-9]+:[a-zA-Z0-9]+$")
+
 
 class LockfileMetadata(BaseModel):
```

Comment:
Just a small, non-blocking suggestion:

It might be worth to add a `extra="forbid"` option here. I was doing a test with an outdated lockfile, and the presence of the `target` attribute did not trigger any error.

Reply:
Added.